yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -644,11 +655,15 @@ def run(self, info):
         input_files = [filename, *sub_filenames]
```

Comment:
`json_filenames` is currently not being included here so no JSON is actually being embedded

Reply:
Hmm gotcha... then why is Grub4K's test failing?